src/main/deltachat.js

```diff
@@ -285,25 +285,25 @@ class DeltaChatController {
   }
 
   selectChat(chatId) {
-    log('selecting chat with id', chatId)
+    log('selecting chat with id', chatId, this)
     this._selectedChatId = chatId
-    electron.ipcMain.emit('render')
-    log(electron)
+    this._render()
   }
 
   /**
    * Returns the state in json format
    */
   render () {
     let chats = this._chats()
+    console.log('render')
```

Comment:
I added this before:

https://github.com/deltachat/deltachat-desktop/blob/40bed31b8d12b188b94e1d6b2aeff00b66b45c41/src/main/ipc.js#L94